packages/commerce-sdk-react/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@salesforce/commerce-sdk-react",
-  "version": "2.0.0-preview.1",
+  "version": "2.0.0",
```

Comment:
Please verify that the changes in this PR are only removing the `-preview.1` from the version numbers.